tests/e2e/e2e_coredaos_test.go

```diff
@@ -0,0 +1,164 @@
+package e2e
+
+import (
+	"fmt"
+	"strconv"
+
+	coredaostypes "github.com/atomone-hub/atomone/x/coredaos/types"
+	govtypesv1 "github.com/atomone-hub/atomone/x/gov/types/v1"
+	govtypesv1beta1 "github.com/atomone-hub/atomone/x/gov/types/v1beta1"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+)
+
+func (s *IntegrationTestSuite) testCoreDAOs() {
+	valIdx := 0
+	chainAAPIEndpoint := fmt.Sprintf("http://%s", s.valResources[s.chainA.id][0].GetHostPort("1317/tcp"))
+	senderAddress, _ := s.chainA.validators[valIdx].keyInfo.GetAddress()
+
+	params := s.queryCoreDAOsParams(chainAAPIEndpoint)
+
+	steeringDAOAddress, err := s.chainA.multiSigAccounts[0].keyInfo.GetAddress()
+	s.Require().NoError(err)
+	oversiteDAOAddress, err := s.chainA.multiSigAccounts[1].keyInfo.GetAddress()
+	s.Require().NoError(err)
+	params.Params.SteeringDaoAddress = steeringDAOAddress.String()
+	params.Params.OversightDaoAddress = oversiteDAOAddress.String()
+	s.writeCoreDAOsParamChangeProposal(s.chainA, params.Params)
+	// Gov tests may be run in arbitrary order, each test must increment proposalCounter to have the correct proposal id to submit and query
+	proposalCounter++
+	submitGovFlags := []string{configFile(proposalParamChangeFilename)}
+	depositGovFlags := []string{strconv.Itoa(proposalCounter), s.queryGovMinDeposit(chainAAPIEndpoint).String()}
+	voteGovFlags := []string{strconv.Itoa(proposalCounter), "yes"}
+	s.submitGovProposal(chainAAPIEndpoint, senderAddress.String(), proposalCounter, "atomone.coredaos.v1.MsgUpdateParams", submitGovFlags, depositGovFlags, voteGovFlags, "vote", govtypesv1beta1.StatusPassed)
+
+	newParams := s.queryCoreDAOsParams(chainAAPIEndpoint)
+	s.Require().Equal(newParams.Params.SteeringDaoAddress, steeringDAOAddress.String())
+	s.Require().Equal(newParams.Params.OversightDaoAddress, oversiteDAOAddress.String())
+
+	s.execBankMultiSend(s.chainA, valIdx, senderAddress.String(),
+		[]string{steeringDAOAddress.String(), oversiteDAOAddress.String()}, tokenAmount.String(), false)
+
+	s.execBankMultiSend(s.chainA, valIdx, senderAddress.String(),
+		[]string{steeringDAOAddress.String(), oversiteDAOAddress.String()}, sdk.NewInt64Coin(uphotonDenom, 100_000_000).String(), false)
```

Comment:
`send` and `multisend` can take multiple `sdk.Coin`. So you can probably call `s.execBankMultiSend` a single time.

Something like this:

```go
s.execBankMultiSend(s.chainA, valIdx, senderAddress.String(),
  []string{steeringDAOAddress.String(), oversiteDAOAddress.String()}, 
  sdk.NewCoins(
    tokenAmount,
    sdk.NewInt64Coin(uphotonDenom, 100_000_000),
  ).String(), 
  false,
)
```